frontend/components/global/BaseDialog.vue

```diff
@@ -178,6 +178,7 @@ export default defineComponent({
     close() {
       this.dialog = false;
       this.logDeprecatedProp("close");
+      this.$emit("closed");
```

Comment:
This emits `closed`, but in edit.vue you've got `@close="resetDialog()"`.. I'm assuming there's an issue caused by the discrepancy in close vs closed?

Reply:
It was just a typo, but it looks like there's already "close" emitted so the above change is unneeded.